package/MDAnalysis/analysis/hydrogenbonds/__init__.py

```diff
@@ -26,3 +26,4 @@
 ]
```

Comment:
Also add to `__all__`?
```suggestion
    'WaterBridgeAnalysis'
]
```